test/e2e/catalog_e2e_test.go

```diff
@@ -214,7 +214,7 @@ var _ = Describe("Starting CatalogSource e2e tests", func() {
 			return err
 		}).Should(Succeed())
 
-		_, err = awaitCSV(crc, generatedNamespace.GetName(), mainCSV.GetName(), csvSucceededChecker)
+		_, err = fetchCSV(crc, mainCSV.GetName(), generatedNamespace.GetName(), csvSucceededChecker)
```

Comment:
FWIW `namespace, name` is the most common & understood order.

Reply:
I agree re: operand order. The problem here is that there are WAY MORE calls to `fetchCSV` than to `awaitCSV` which is why `s/awaitCSV/fetchCSV/g` occurred, rather than the other way around.

Reply:
Originally:
* awaitCSV: 49
* fetchCSV: 113

Reply:
`sed` is my friend.